model/Check/System/TaskQueueFinishedCheck.php

```diff
@@ -109,48 +115,41 @@ private function getTasksStatistics() : array
 
         $result = [];
         foreach (self::OPTIONS_INTERVAL as $name => $interval) {
-            $intervalObj = new \DateInterval($interval);
-            $period = new \DateInterval($name);
+            $intervalObj = new DateInterval($interval);
+            $period = new DateInterval($name);
 
             $points = round($this->getIntervalSeconds($period) / $this->getIntervalSeconds($intervalObj));
```

Comment:
I would ask you to split this line into 2 or 3 lines, with an `if` block. Reason: even if most of the code below was changed, the function called in this line was not - there is still space for `division by zero`.

Reply:
done